projects/kit/src/lib/utils/date/segments-to-date.ts

```diff
@@ -17,7 +17,7 @@ export function segmentsToDate(
     );
 
     // needed for years less than 1900
-    date.setUTCFullYear(Number(year ?? '0'));
+    date.setFullYear(Number(year ?? '0'));
```

Comment:
Could you please change PR's title to correctly appears in changelog (it is now difficult to understand which bug it fixes) ?